test/parallel/sdk.API.submitableExtrinsic.test.ts

```diff
@@ -0,0 +1,321 @@
+    [defaultCurrencyValue],
+    sudo
+  );
+
+  await Sudo.batchAsSudoFinalized(
+    Assets.mintNative(testUser),
+    Assets.mintToken(token2, testUser, Assets.DEFAULT_AMOUNT)
+  );
+
+  const tx = await mangata.submitableExtrinsic.createPool({
+    account: testUser.keyRingPair.address,
+    firstTokenAmount: Assets.DEFAULT_AMOUNT.divn(2),
+    firstTokenId: MGA_ASSET_ID.toString(),
+    secondTokenAmount: Assets.DEFAULT_AMOUNT.divn(2),
+    secondTokenId: token2.toString(),
+  });
+
+  await signSubmittableExtrinsic(tx, testUser);
+
+  const balancePool = await getBalanceOfPool(MGA_ASSET_ID, token2);
+  const liqId2 = await getLiquidityAssetId(MGA_ASSET_ID, token2);
+
+  expect(liqId2).bnGt(BN_ZERO);
+  expect(balancePool[0]).bnEqual(Assets.DEFAULT_AMOUNT.divn(2));
+  expect(balancePool[1]).bnEqual(Assets.DEFAULT_AMOUNT.divn(2));
+});
+
+test("check multiswapBuyAsset", async () => {
+  const tx = await mangata.submitableExtrinsic.multiswapBuyAsset({
+    account: testUser.keyRingPair.address,
+    tokenIds: [MGA_ASSET_ID.toString(), token1.toString()],
+    amount: BN_HUNDRED,
+    maxAmountIn: BN_BILLION,
+  });
+
+  await testUser.refreshAmounts(AssetWallet.BEFORE);
+
+  await signSubmittableExtrinsic(tx, testUser);
+
+  await testUser.refreshAmounts(AssetWallet.AFTER);
+
+  const amountDifference = testUser
+    .getAsset(token1)
+    ?.amountAfter.free!.sub(testUser.getAsset(token1)?.amountBefore.free!);
+
+  expect(amountDifference).bnEqual(BN_HUNDRED);
+});
+
+test("check multiswapSellAsset", async () => {
+  const tx = await mangata.submitableExtrinsic.multiswapSellAsset({
+    account: testUser.keyRingPair.address,
+    tokenIds: [MGA_ASSET_ID.toString(), token1.toString()],
+    amount: BN_HUNDRED,
+    minAmountOut: BN_ONE,
+  });
+
+  await testUser.refreshAmounts(AssetWallet.BEFORE);
+  await signSubmittableExtrinsic(tx, testUser);
+
+  await testUser.refreshAmounts(AssetWallet.AFTER);
+
+  const amountDifference = testUser
+    .getAsset(token1)
+    ?.amountAfter.free!.sub(testUser.getAsset(token1)?.amountBefore.free!);
+
+  expect(amountDifference).bnGt(BN_ZERO);
+  expect(amountDifference).bnLte(BN_HUNDRED);
+});
+
+test("check transferTokens", async () => {
+  const [testUser1] = setupUsers();
+
+  const tx = await mangata.submitableExtrinsic.transferTokens({
+    account: testUser.keyRingPair.address,
+    address: testUser1.keyRingPair.address,
+    amount: BN_BILLION,
+    tokenId: MGA_ASSET_ID.toString(),
+  });
+
+  await signSubmittableExtrinsic(tx, testUser);
+
+  const balance = await getUserBalanceOfToken(MGA_ASSET_ID, testUser1);
+
+  expect(balance.free).bnEqual(BN_BILLION);
+});
+
+test("check transferAllTokens", async () => {
+  const [testUser1] = setupUsers();
+
+  const tx = await mangata.submitableExtrinsic.transferAllTokens({
+    account: testUser.keyRingPair.address,
+    address: testUser1.keyRingPair.address,
+    tokenId: token1.toString(),
+  });
+
+  await signSubmittableExtrinsic(tx, testUser);
```

Comment:
can you check the events for one of the submissions?

Reply:
SignSubmittableExtrinsic returns the events related to this extrinsic. We are not testing that explicitly, so it would be awesome to have some validations that ensure that the expected events are returned correctly. 
What do you think?

Reply:
Sure, added getEventResultFromMangataTx